helpers/bot_client.py

```diff
@@ -290,6 +290,22 @@ async def flip(self, ctx, number=1):
         except Exception as e:
             await ctx.send(str(e))
 
+    @commands.command(name='roll', help='Rolls a dice of your choosing')
+    async def roll(self, ctx, number=1, dice_size="D6"):
```

Comment:
nit - rename `number` variable to be something more descriptive, i.e. `num_rolls` or `dice_rolls`

Reply:
Additionally, change the double quotes wrapping `D6` to single quotes